pyaedt/generic/constants.py

```diff
@@ -11,6 +11,7 @@
 METER2IN = 0.0254
 METER2MILES = 1609.344051499
 
+MILS2METER = 39370.078740157
```

Comment:
Are you sure that MILS2METER = 39370.078740157
I thought that 1 mils = 2.54e-5 meters.

Reply:
it's the inverse of it